addon/components/o-s-s/information-section.hbs

```diff
@@ -0,0 +1,41 @@
+<div
+  data-control-name="information-section"
+  class="oss--information-section border-radius-md border-color-gray-200 box-shadow-md"
+  ...attributes
+>
+  <div
+    class="fx-row fx-malign-space-between fx-xalign-center padding-px-18 {{this.plainClass}}"
+    data-control-name="information-section--header"
```

Comment:
```suggestion
    data-control-name="information-section-header"
```
We never use double dash in `data-control-name`, maybe just use single dash

Reply:
Cleaned up